AK/JsonObject.h

```diff
@@ -47,10 +47,16 @@ class JsonObject {
     int size() const { return m_members.size(); }
     bool is_empty() const { return m_members.is_empty(); }
 
-    JsonValue get(String const& key) const
+    JsonValue const& get(String const& key) const
     {
         auto* value = get_ptr(key);
-        return value ? *value : JsonValue(JsonValue::Type::Null);
+        static JsonValue* s_null_value { nullptr };
+        if (!value) {
+            if (!s_null_value)
+                s_null_value = new JsonValue;
```

Comment:
Might be nicer to not allocate it from the heap?

Reply:
Let's just leave it as is, it's quite annoying to try to get it to work otherwise.